persist/fs/manager.go

```diff
@@ -37,16 +37,15 @@ type sleepFn func(time.Duration)
 // persistManager is responsible for persisting series segments onto local filesystem.
 // It is not thread-safe.
 type persistManager struct {
-	opts                    Options
-	filePathPrefix          string
-	throughputCheckInterval time.Duration
-	throughputLimitMbps     float64
-	nowFn                   clock.NowFn
-	sleepFn                 sleepFn
-	writer                  FileSetWriter
-	start                   time.Time
-	lastCheck               time.Time
-	bytesWritten            int64
+	opts                Options
+	filePathPrefix      string
+	throughputLimitOpts persist.ThroughputLimitOptions
```

Comment:
I wonder if we should put `ThroughputLimitOptions` into `x/io` so that when we need to use something very similar for network throughput limiting we can reuse it?